### PR TITLE
adapt aardvark adapter for indexexchange

### PR DESCRIPTION
**Pull request by dejanstrbac**

### CODE REVIEW

**Comment by ix-certification** [comment]
Please resolve the merge conflicts.

**Comment by dejanstrbac** [comment]
rebased and fixed according to your requirements

File: rtk-htb.js [hunk 1]
```diff
@@ -144,9 +144,36 @@ function RtkHtb(configs) {
         /* ---------------------- PUT CODE HERE ------------------------------------ */
```
Comment (ix-certification) [inline rtk-htb.js]: Please consult #9 for more information about updating your __profile variable

File: rtk-htb.js [hunk 2]
```diff
@@ -144,9 +144,36 @@ function RtkHtb(configs) {
...
         var queryObj = {};
         var callbackId = System.generateUniqueId();
+        var auctionCode = false,
```
Comment (ix-certification) [inline rtk-htb.js]: Please consult #9 for more information about updating your __profile variable

File: rtk-htb.js [hunk 3]
```diff
@@ -176,9 +203,22 @@ function RtkHtb(configs) {
         var privacyEnabled = ComplianceService.isPrivacyEnabled();
 
         /* ---------------- Craft bid request using the above returnParcels --------- */
+        queryObj.jsonp = 'window.' + SpaceCamp.NAMESPACE + '.' + __profile.namespace +
```
Comment (ix-certification) [inline rtk-htb.js]: This can be removed if your endpoint uses ajax rather than jsonp

**Pull request merged**